### PR TITLE
Some Update for VM backup, snapshot and restore

### PR DESCRIPTION
**Pull request by WebberHuang1118**
Some update for VM Backup, Snapshot and Restore

Related issue:
#373

### CODE REVIEW

File: versioned_docs/version-v1.1/vm/backup-restore.md [hunk 1]
```diff
@@ -140,7 +142,9 @@ VM snapshots are created from the **Virtual Machines** page. The VM snapshot vol
 
 **Result:** The snapshot is created. You can also go to the `Backup & Snapshot > VM Snapshots` page to view all VM snapshots.
 
-The `ReadyToUse` status will be set to `true` once the snapshot is complete.
+The `State` will be set to `Ready` once the snapshot is complete.
```
Comment (guangbochen) [inline versioned_docs/version-v1.1/vm/backup-restore.md:145]: @WuJun2016 can you please help to double-check if this is true for v1.1, thanks.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks for the update.

File: versioned_docs/version-v1.1/vm/backup-restore.md (hunk 1 above)
Comment (WuJun2016) [reply versioned_docs/version-v1.1/vm/backup-restore.md:145]: They also apply to v1.1.

File: docs/vm/backup-restore.md [hunk 2]
```diff
@@ -75,8 +77,8 @@ You can choose to either delete or retain the previous volumes. By default, all
 
 1. Go to the `VM Backups` page.
 1. Click `Replace Existing`.
-
-The restore process can be viewed from the `Virtual Machines` page.
+1. The restore process can be viewed from the `Virtual Machines` page.
```
Comment (LucasSaintarbor) [inline docs/vm/backup-restore.md:80]:
```suggestion
1. You can view the restore process from the `Virtual Machines` page.
```

File: docs/vm/backup-restore.md [hunk 3]
```diff
@@ -165,5 +169,5 @@ You can only choose to retain the previous volumes.
 
 1. Go to the `VM Snapshots` page.
 1. Click `Replace Existing`.
-
-The restore process can be viewed from the `Virtual Machines` page.
+1. The restore process can be viewed from the `Virtual Machines` page.
```
Comment (LucasSaintarbor) [inline docs/vm/backup-restore.md:172]:
```suggestion
1. You can view the restore process from the `Virtual Machines` page.
```

File: versioned_docs/version-v1.1/vm/backup-restore.md [hunk 4]
```diff
@@ -75,8 +77,8 @@ You can choose to either delete or retain the previous volumes. By default, all
 
 1. Go to the `VM Backups` page.
 1. Click `Replace Existing`.
-
-The restore process can be viewed from the `Virtual Machines` page.
+1. The restore process can be viewed from the `Virtual Machines` page.
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.1/vm/backup-restore.md:80]:
```suggestion
1. You can view the restore process from the `Virtual Machines` page.
```

File: versioned_docs/version-v1.1/vm/backup-restore.md [hunk 5]
```diff
@@ -165,5 +169,5 @@ You can only choose to retain the previous volumes.
 
 1. Go to the `VM Snapshots` page.
 1. Click `Replace Existing`.
-
-The restore process can be viewed from the `Virtual Machines` page.
+1. The restore process can be viewed from the `Virtual Machines` page.
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.1/vm/backup-restore.md:172]:
```suggestion
1. You can view the restore process from the `Virtual Machines` page.
```

**Review by LucasSaintarbor — CHANGES_REQUESTED** [review]
Small NITs. Otherwise, LGMT 👍

File: docs/vm/backup-restore.md (hunk 2 above)
Comment (WebberHuang1118) [reply docs/vm/backup-restore.md]: Updated per suggestion. Thanks.

File: docs/vm/backup-restore.md (hunk 3 above)
Comment (WebberHuang1118) [reply docs/vm/backup-restore.md]: Updated per suggestion. Thanks.

File: versioned_docs/version-v1.1/vm/backup-restore.md (hunk 4 above)
Comment (WebberHuang1118) [reply versioned_docs/version-v1.1/vm/backup-restore.md]: Updated per suggestion. Thanks.

File: versioned_docs/version-v1.1/vm/backup-restore.md (hunk 5 above)
Comment (WebberHuang1118) [reply versioned_docs/version-v1.1/vm/backup-restore.md]: Updated per suggestion. Thanks.

**Comment by guangbochen** [comment]
@WebberHuang1118 u may need to rebase the PR in order to pass the CI as we have introduced new changes in CI recently, thanks.

**Comment by WebberHuang1118** [comment]
> @WebberHuang1118 u may need to rebase the PR in order to pass the CI as we have introduced new changes in CI recently, thanks.

Got it, many thanks.

**Review by WebberHuang1118 — COMMENTED** [review]
Updated per suggestion, thanks.

**Review by LucasSaintarbor — APPROVED** [review]
LGTM, thanks

**Pull request merged**